src/controllers/update/beneficial-owner-bo-mo-review.controller.ts

```diff
@@ -0,0 +1,38 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+import { ApplicationData } from "model";
+import { getApplicationData } from "../../utils/application.data";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+    const appData: ApplicationData = getApplicationData(req.session);
+
+    return res.render(config.BENEFICIAL_OWNER_BO_MO_REVIEW_PAGE, {
+      // TODO temporary PAGE for testing, remove afterwards
```

Comment:
can update now that previous page is in.

Reply:
Done.